clowdapp.yaml

```diff
@@ -0,0 +1,190 @@
+---
+
+apiVersion: v1
```

Comment:
I'm a little confused by the two apiVersions present in this file, both v1 and v1alpha1.  Are you at times referring to like our views and openapi spec for the regular api and at other times referring to the new clowder app or should all of these be consistently one name (lines 3, 9, 18, 125, 145, 153)?

Also, if you have a cloudl.redhat.com/v1alpha1 endpoint, don't you need to also have views for this like we have for regular, internal, and mgmt already?

Reply:
I am not sure but isn't this version of the clowder protocol? For the format of the content of this file? Just asking...

Reply:
Hey @CodeHeeler, this is not related to our endpoints and services it self.

It's a specific Clowder configuration pattern that is specified in the docs (need to be in VPN):

https://clouddot.pages.redhat.com/clowder/dev/api_reference.html#k8s-api-github-com-redhatinsights-clowder-apis-cloud-redhat-com-v1alpha1-clowdapp

Here an example from the Clowder docs:

https://clouddot.pages.redhat.com/clowder/dev/usage/jobs.html

Thanks